datumaro/plugins/datumaro_format/extractor.py

```diff
@@ -190,6 +191,18 @@ def _load_annotations(item):
         return loaded
 
 class DatumaroImporter(Importer):
+    @classmethod
+    def detect(cls, context: FormatDetectionContext) -> None:
+        annot_file = context.require_file('annotations/*.json')
+
+        with context.probe_text_file(
+            annot_file, "must be a JSON object with \"categories\" "
+                "and \"items\" keys",
+        ) as f:
+            contents = json.load(f)
+            if not {'categories', 'items'} <= contents.keys():
```

Comment:
Maybe we should check there is no `info` field to avoid conflicting with coco.

Reply:
> Maybe we should check there is no info field to avoid conflicting with coco.

I don't think there is a conflict, otherwise the COCO detection test would have failed (since there is currently no medium-confidence detector for COCO, it would have gotten detected as Datumaro instead).

> Think if we can use https://github.com/ICRAR/ijson to avoid storing big jsons in memory.

Interesting... I'll investigate it.